src/components/Sponsors/Sponsorcard.tsx

```diff
@@ -0,0 +1,17 @@
+import { Sponsor } from './config';
+
+interface SponsorCardProps {
+  sponsor: Sponsor;
+}
+
+const SponsorCard: React.FC<SponsorCardProps> = ({ sponsor }) => {
+  return (
+    <div className="sponsor mx-auto rounded-lg bg-gray-900 shadow-lg p-5 flex flex-col max-w-md lg:max-w-xl">
```

Comment:
The width should be increased
The background colour does not match:

![image](https://github.com/theDS3/Website/assets/32755895/68c04dc8-2a17-4b68-949f-31d95be6f837)
![image](https://github.com/theDS3/Website/assets/32755895/eef7e1cc-b1ca-4d24-9502-d8c488bc00ce)

Reply:
For TailwindCSS arbitrary colors: https://tailwindcss.com/docs/customizing-colors#arbitrary-values